src/typhoon_automator/automator.py

```diff
@@ -213,8 +223,48 @@ def save_scenarios(
     def run(
             self,
             use_vhil: bool = False):
+        """ Run all scenarios in the automation
+
+        :param bool use_vhil:
+        """
+        
         self._model.load_to_setup(use_vhil=use_vhil)
-        raise NotImplementedError()
+        
+        if self._orchestrator is None:
+          raise RuntimeError("Automation is not initialized")
+    
+        start_time = datetime.now()
+        self._logger.info(f"Starting scenario simulations at {start_time.strftime('%H:%M:%S, %m/%d/%Y')}")
```

Comment:
There's a new logging method in the automator.  Logging info from the automator can now simply be: ```self.log(f"Super helpful information")```.  Check out the log method near the top of the class.